day-four-challenge/secure-container/getPasswords.js

```diff
@@ -1,34 +1,69 @@
-const { map, sort, not, uniq, equals, unfold, allPass, pipe } = require('ramda');
+const { map, sort, not, uniq, allPass, join, replace, range, subtract } = require('ramda');
 
 const MIN = 359282;
 const MAX = 820401;
-const PASSWORD_MIN_LENGTH = 6;
-
-const diff = (a, b) => { return a - b };
+const PASSWORD_LENGTH = 6;
 
+/** Determines if an array is sorted
```

Comment:
Putting the description on the second line is more in-line with the standard: https://jsdoc.app/about-getting-started.html
```suggestion
/** 
 * Determines if an array is sorted
```